sources/platform/console/settings.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Settings
+description: Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.
+sidebar_position: 3
+category: platform
+slug: /console/settings
+---
+
+**Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.**
+
+---
+
+## Account
+
+By clicking the **Settings** tab on the side menu, by default, you will be presented with an Account page where you can view & edit various settings regarding your account, such as:
+
+* account email
+* username
+* profile information
+* theme
+* login information
+* account delete
+
+## Integrations
+
+The Integrations page provides essential tools for enhancing your interaction with our platform. Here, you can access your **Personal API Tokens**, which are necessary for using our [REST API](https://docs.apify.com/api/v2). This page also facilitates the integration of your Slack workspace and lists your **Actor Integration Accounts**. This section represents any third-party integrations added to Actors developed by you or your team. For detailed guidance on utilizing these integrations, refer to our [Integrations documentation](https://docs.apify.com/platform/integrations).
```

Comment:
Question: Elsewhere we're using `tab`. Should we call it the same in all parts of the docs?

Reply:
Suggestion: Maybe we should say `Apify Platform` instead of `our platform`?

Reply:
And last one: 
```suggestion
The Integrations page provides essential tools for enhancing your interaction with our platform. Here, you can access your **Personal API Tokens**, which are necessary for using our [REST API](https://docs.apify.com/api/v2). This page also facilitates the integration of your Slack workspace and lists your **Actor Integration Accounts**. This section represents any third-party integrations added by you or your team. For detailed guidance on utilizing these integrations, refer to our [Integrations documentation](https://docs.apify.com/platform/integrations).
```